### PR TITLE
Added some extra installation extra steps based on cob4-5 installation

### PR DESCRIPTION
**Pull request by ipa-nhg**
The extra installation should be integrated on the installation script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,131 @@
...
+  * Setup udev rules
+  * Setup bash environment 
+
++ **Setup NTP and NFS** This option will configure the NFS system depending on the installation mode, it is important that the master pc is already installed and per network reachable before install the slave computers, otherwise the installation process will be cancelled. After this installation it is necessary restart the computer.
+ 
++ **Full Installation** A full installation means the combination of 1 and 2 (Basic Installation + Setup NTP and NFS)
++ **Cob setup** This step holds the recommended configuration of the robot home directory. This step can only be execute after a full installation.
+
+### 2. Extra Installation <a id="Extra-Installation"/> 
+#### 2.1. Hands configuration <a id="Hands"/>
```
Comment (ipa-nhg) [inline README.md]: @ipa-mdl FYI

**Comment by ipa-fmw** [comment]
FR: OK

**Comment by ipa-nhg** [comment]
New commit : https://github.com/ipa320/setup_cob4/pull/34/commits/e11b34bc0f006021f366652b7ca387d9da08a07d 
@ipa-fmw Added administrator manual , with the subsections: network setup, creation of new users and backup.

**Comment by ipa-fmw** [comment]
Admin manual is good. 

I'd like to separate this into a standalone README for installation and admin, and then link to it from the main README (which should be very short and only list intallation and admin manual links so far).

**Comment by ipa-fmw** [comment]
FR: OK

**Pull request merged**